### ISSUE TITLE
Jupyter Notebook for Experiments

### ISSUE THREAD

**Issue by Simsso** [Simsso/NIPS-2018-Adversarial-Vision-Challenge#24]
Create a Jupyter notebook that can be used to quickly try attacks, compute gradients, and play around in general.

Maybe we should have (1) one model for MNIST (for super fast training convergence and experiment iteration; [here](https://github.com/ianlewis/tensorflow-examples/blob/master/notebooks/TensorFlow%20MNIST%20tutorial.ipynb) is an inspiration) and (2) one for Tiny ImageNet, just to have some _more realistic_ testing platform as well. The latter could be based on [the TF cifar10 network](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10).
